src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -797,6 +797,11 @@ bool is_supported_pad(const layout& layout) {
         no_spatial_padding &= (pad._upper_size[2 + i] == 0);
     }
 
+    // Check feature padding
```

Comment:
This change also disables the ability to use feature padding for other primitives
It's expected? Maybe restrict the conditions only for reorder in src/plugins/intel_gpu/src/graph/impls/onednn/reorder_onednn.hpp?